blueprints/edition-detector.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = function(blueprint) {
+  blueprint.filesPath = function() {
+    let rootPath = process.env.EMBER_VERSION === 'OCTANE' ? 'native-files' : 'files';
```

Comment:
Wasn't this edition in the blueprint? Am I misremembering?

Reply:
I don't understand the question.

Here is checking if the `ENV.EMBER_VERSION` is equal to 'OCTANE` to use Native files instead of Ember `extend`.

https://github.com/ember-cli/ember-octane-blueprint/pull/36 set the env variable.

Reply:
ah ok, you right. I was misremembering. thank you! :)